api/src/test/java/org/openmrs/module/dhisconnector/api/DHISConnectorServiceTest.java

```diff
@@ -186,4 +186,36 @@ public void testDHISMappingExport() throws IOException {
 				dhisConnectorService.exportMappings(new String[]{"mapping-test-unit." + mapping.getCreated()});
 		Assert.assertEquals("Successfully bundled the mapping with the metadata", pathToBundle[0]);
 	}
+
+	@Test
+	public void testDHISMappingImport() throws IOException {
+		DHISConnectorService dhisConnectorService = Context.getService(DHISConnectorService.class);
+		ReportDefinitionService reportDefinitionService = Context.getService(ReportDefinitionService.class);
+
+		DHISMapping mapping = new DHISMapping();
+		mapping.setName("mapping-test-unit");
+		mapping.setCreated(new Date().getTime());
+		PeriodIndicatorReportDefinition pir = new PeriodIndicatorReportDefinition();
+		UUID pirUuid = UUID.randomUUID();
```

Comment:
Is this a constant?

Reply:
uuid?

Reply:
tru dat!

Reply:
I'm trying to generate a random UUID to make a test period indicator report

Reply:
no

Reply:
So, is it a constant?

Reply:
yes 🙂 do you want me to mark it as static?

Reply:
Use your programming knowledge to make the best judgement. 😊

Reply:
Sure 😄 
You're a good reviewer

Reply:
I marked it final